spec/dummy/config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,13 @@
+sidekiq_connection = {
+  url: Settings.sidekiq.url,
```

Comment:
I didn't found this settings explicite configure. Would like to show in which file may I put excplicite sidekiq settings?